clab/exec/exec.go

```diff
@@ -32,25 +30,23 @@ func ParseExecOutputFormat(s string) (ExecOutputFormat, error) {
 	return "", fmt.Errorf("cannot parse %q as 'ExecOutputFormat'", s)
 }
 
-type ExecCmd interface {
-	GetCmd() []string
-	GetCmdString() string
-}
```

Comment:
let me merge it rn without this interface, since I want to carve a xmas release. And then we re-iterate, since I don't see a need for ResultHolder interface either.